image/v1/types.go

```diff
@@ -605,6 +605,10 @@ type ImageBlobReferences struct {
 	// not have separate config blobs and this field will be set to nil if so.
 	// +optional
 	Config *string `json:"config" protobuf:"bytes,2,opt,name=config"`
+	// manifests is the list of manifests that compose this image. For a
```

Comment:
What is the format of a manifest that can be returned?  Please provide an example.

Reply:
rephrased it, it's a list of image names.